lib/auth.ts

```diff
@@ -123,10 +127,11 @@ export class AuthSdk {
 				throw new Error('Token has expired');
 			}
 
-			return token;
+			return sessionId;
```

Comment:
This change makes the commit a breaking change. I also think this should possibly be returning an object containing both the session ID and the session token, I could be wrong about this though, it depends on what you think the implementing code should do

Reply:
Previously, the SDK was referring to the session ID as `token` so, unless I'm missing something, this method is returning the exact same variable, just with a more descriptive (?) name. The only thing that changed was the method's signature, which now includes an optional argument `token`. If this argument is not provided, the `authToken` attribute is set to `null`, and the client defaults to using the session ID for authentication, thus ensuring compatibility. The idea was to eventually make `token` mandatory, and have everyone using the SDK adapt to this change. Does this make sense?